### DISCUSSION TITLE
Failed Vercel Deployment

### DISCUSSION THREAD

**Discussion by kisa5** [neuefische/questions#461] [Web] [answered]
Somehow my **Vercel Deployment on my feature branch** fails. I tried one or two things suggested on Stackoverflow but it didn't worked. I have done some search for similar discussions here on the github discussion board but it did not word for me either.

<img width="827" alt="Bildschirmfoto 2023-01-23 um 14 29 03" src="https://user-images.githubusercontent.com/116880659/214066815-ced9c67b-e46c-400f-bfeb-d93f68ad695b.png">

Here a link to my [PR](https://github.com/kisa5/capstone-project/pull/3) also with the error message.

**Comment by mpagels** [comment]
The `build` process runs through locally with two changes in `./component/ProjectItem.js`

change the import line to
```diff
- import { titles } from "lib/db.json";
+ import titles from "lib/db.json";
```

and the map function to
```diff
- {titles.map((title) => (
+ {titles.titles.map((title) => (
```

**Comment by mbosselmann** [comment]
Hi Anna,
I think I found the problem: The name of your ProjectItem component is not `ProjectItem`, but `ProjectCard` (see your `ProjectItem.js`). But you import a `ProjectItem` component in your `pages/index.js`: `import ProjectItem from "components/ProjectItem";` `ProjectItem` in your `ProjectItem.js` is currently a styled component.  Something with the naming went wrong here. :)

I hope this does help!

**Reply by mpagels** [reply]
If I'm not wrong, the export of the function in the `ProjectItem.js` file is a `default` one. So no problem using a different name in the `import` statement. Read more about it [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/export?retiredLocale=de#description) (scroll down a bit to the `// file test.js` example)

**Reply by mbosselmann** [reply]
So true! You are right. I think I was confused about the vercel error message. 😓

**Reply by mpagels** [reply]
I get you. The vercel error message is not very helpful. I cloned the project and build the project locally. The next build error message is a bit clearer what the problem might be.

**Comment by kisa5** [comment, marked as answer]
Thank you for your help! I fixed the problem by deleting the babelrc file and correcting the import from the logo.svg.  😅
